src/screens/ContactScreen/index.js

```diff
@@ -93,26 +97,28 @@ const ContactScreen = ({navigation}) => {
     const initialData = await DiscoveryRepo.fetchInitialDiscoveryUsers(
       50,
       parseInt(userPage.currentPage, 10),
-      isAnon
+      isAnon,
+      {
+        excludedUserNames: existParticipants
+      }
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The function call to `fetchInitialDiscoveryUsers` now includes an additional parameter which is an object containing `excludedUserNames`. This seems to be a way to exclude certain users from the discovery process. However, it's important to ensure that the `fetchInitialDiscoveryUsers` function properly handles this new parameter.
```diff
-      isAnon
+      isAnon,
+      {
+        excludedUserNames: existParticipants
+      }
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->